src/metatensor/models/utils/data/dataset.py

```diff
@@ -15,6 +15,7 @@ class TargetInfo:
     :param quantity: The quantity of the target.
     :param unit: The unit of the target.
     :param per_atom: Whether the target is a per-atom quantity.
+    :param gradients: The gradients of the target.
```

Comment:
That's not quite enough information ^_^. Where are these gradients stored? what is the model supposed to do with them?